src/platforms/hosted/bmp_libusb.c

```diff
@@ -19,13 +19,72 @@
 
 /* Find all known usb connected debuggers */
 #include "general.h"
+#if defined(_WIN32) || defined(__CYGWIN__)
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#include <stdint.h>
+
+#include "ftd2xx.h"
+#else
 #include <libusb.h>
+#include "libftdi1/ftdi.h"
```

Comment:
`pkg-config --cflags libftdi1` adds `/usr/include/libftdi1` to the include paths, meaning including the directory in this is wrong, especially if that containing directory name is different for any reason on someone's system. This should be more correct:
```suggestion
#include <ftdi.h>
```

Reply:
Sugested edit applied.